src/services/game-logic/Board.ts

```diff
@@ -27,6 +27,14 @@ class Board {
         return this.state[square.row][square.column];
     }
 
+    public checkAllSquares = (f: any) => {
```

Comment:
czym jet F? jednoliterowe zmienne to zuo. 'callback' brzmi lepiej, a jeszcze lepiej jak od razu wiadomo co ten callback ma robić :)

Reply:
poprawiłam :)